storybook/index.html

```diff
@@ -374,13 +374,35 @@ <h1 class="sectionHeading">BAMP</h1>
                 </section>
 
                 <section class="demo" data-module-name="muralThumbnail">
-                    <button>
-                        <div class="box box-red">
-                            <div class="muralThumbnail--img"><img src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcS73Kk1K63Laa0bWG-7nwb7gCl4hKxoxeqfRw&usqp=CAU"></div>
-                            <div class="muralThumbnail--location">Oakland Ca</div>
-                            <div class="muralThumbnail--name">I dont Know</div>
+                    <div class="gridBox gridBox-col3 gridBox-center">
+                        <div class="muralBox">
+                            <button id="openMuralModal">
```

Comment:
ID's can only be used once, so this will have to be a class -- and maybe we can just reuse the "muralThumbnail" class for targeting.

I think the ideal HTML would look like this:

```html
<button class="muralThumbnail box box-red" data-mural-name="lodestar">...</button>
```

...and then you could target it with `document.querySelectorAll(".muralThumbnail")`